src/commands/config.ts

```diff
@@ -12,23 +13,31 @@ import {
 } from "../database";
 import { Bot, BotCommand } from "../structures";
 
-const badges: [string, string][] = Object.keys(DEFAULT_BADGES).map((v) => [
-    v,
-    v,
-]);
-const specRoles: [string, string][] = ["announcements", "members"].map((v) => [
-    v,
-    v,
-]);
-const specChannels: [string, string][] = [
+const badges: APIApplicationCommandOptionChoice<string>[] = Object.keys(
+    DEFAULT_BADGES
+).map((v) => ({
+    name: v,
+    value: v,
+}));
+const specRoles: APIApplicationCommandOptionChoice<string>[] = [
+    "announcements",
+    "members",
+].map((v) => ({
+    name: v,
+    value: v,
+}));
+const specChannels: APIApplicationCommandOptionChoice<string>[] = [
```

Comment:
> https://discord.js.org/#/docs/builders/stable/class/SlashCommandStringOption?scrollTo=addChoices

It looks like a [name, value] situation; even if this did accept objects, why is this preferred over what was already there?